docs/evaluate/development-production-features/low-latency.mdx

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This page seems like it's talking about better latency when compared to Temporal on-prem, but it never really calls that out. A naive read of this would be "Temporal provides less latency than just a normal application" which cannot possibly be true as adding Temporal inherently adds latency.